### DISCUSSION TITLE
Logging to syslog server & changing current log path

### DISCUSSION THREAD

**Discussion by DjordjeMandic** [LibreHardwareMonitor/LibreHardwareMonitor#1007] [Ideas]
Since #995 talks about moving settings to AppData should then logs move also to subfolder `Logs` in same folder where settings might be stored? 

Is logging this stuff to syslog even a useful thing? Maybe some servers can filter out specific entries to single file just for LHM if user needs it?

**Comment by smuecc** [comment]
Did you find any solution for the log path?

**Reply by DjordjeMandic** [reply]
This is just a suggestion.  I didn't do anything yet but I might make PR for syslog client.
